search/roadmap.adoc

```diff
@@ -26,34 +26,46 @@ Analyzers::
 A new DSL will be introduced to allow people to define Analyzers programmatically at boostrap, as an alternative to the current `@Analyzer` and `@AnalyzerDef` annotations.
 We might want to automatically register a set of commonly used Analyzers - let us know what you think of this as the choice is hard!
 
+Java 9 compatibibility::
+We're aiming at regularly testing compatibility with Java 9 on top of the regular tests on Java 8.
+Testing will be limited to having Hibernate Search on the regular classpath: running as a module is an additional step we deferred to version 5.9.
+
+
+== Hibernate Search 5.9
+
+Elasticsearch performance::
+We applied some interesting performance improvements in version 5.8 but there are some more interesting ideas which could provide
+a radical boost. Moved these to 5.9 as they are quite complex and need fleshing out.
+
 WildFly Swarm::
```

Comment:
Not sure what's behind this exactly. If we need input from other teams, we might want to avoid putting such goal in 5.9: we don't want to delay the release because of events we can't control.

Reply:
We had promised to do this to the Swarm team, it was originally scheduled for 5.8